.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,43 @@
 <!-- Please review the following before submitting a PR:
 Che's Contributing Guide: https://github.com/eclipse/che/blob/master/CONTRIBUTING.md
 Pull Request Policy: https://github.com/eclipse/che/wiki/Development-Workflow#pull-requests
-
-COMMITTERS: please include labels on each PR. Labels are listed here: https://github.com/eclipse/che/wiki/Labels but at a minimum you should include `kind/..` and `Dev Open Pull Request Status` labels.
 -->
 
 ### What does this PR do?
 
 
+### Screenshot/screencast of this PR
+<!-- Please include a screenshot or a screencast explaining what is doing this PR -->
+
+
 ### What issues does this PR fix or reference?
+<!-- Please include any related issue from eclipse che repository (or from another issue tracker).
+     Include link to other pull requests like documentation PR from [the docs repo](https://github.com/eclipse/che-docs)
+-->
+
+
+### How to test this PR?
+<!-- Please explain for example :
+  - The test platform (openshift, kubernetes, minikube, CodeReady Container, docker-desktop, etc)
+  - Installation method: chectl / che-operator
+  - steps to reproduce
+ -->
+
 
-<!-- #### Changelog -->
-<!-- The changelog will be pulled from the PR's title. 
-     Please provide a clear and meaningful title to the PR and don't include issue number -->
+### PR Checklist
 
+As the author of this Pull Request I made sure that:
 
-#### Release Notes
-<!-- markdown to be included in marketing announcement - N/A for bugs -->
+- [ ] The Eclipse Contributor Agreement is valid
+- [ ] Code produced is complete
+- [ ] Code builds without errors
+- [ ] Add new automated test for every bug fixed
+- [ ] The repository devfile is up to date and works
+- [ ] Sections `What issues does this PR fix or reference` and `How to test this PR` completed
```

Comment:
It's unclear for me why do we need this, as we already have templates for both sections.

Reply:
it's just to not let these sections 'blank'

Reply:
isn't that already explicit ehough? It's like add another checkbox `[ ] all checkboxes above are checked`

Reply:
yes but a lot of PR have no description or how to be able to test it paragraph even if it sounds logical to explain how to describe what the PR is doing and how to validate it

Also on some PR, the template is totally removed so it's to ensure consistency